CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### Added
+
+- Add getters for `Validator` object to expose more data to users.
```

Comment:
want to update version from 0.0.15 -> 0.0.16 ?

Reply:
yeah done!